amida-tech/orange-api
# Task/update deps

#### What does this do
This updates a couple key dependencies that have known vulnerabilities.
See `package.json` diff for the updates.

From what I can tell, the only api breaking update was Twilio and I made the appropriate changes according to https://www.twilio.com/docs/sms/quickstart/node

The only thing that GitHub was complaining about that should still be there is an old `uglify`, which is a subsubsubsubdependency of `aglio`, our documentation generating tool. Specifically, `aglio`s default theme hasn't been updated and relies on `jade`, the deprecated version of `pug`... and judging by activity on the repo, it doesn't look like this will be fixed any time soon. However, this is fine to keep as a dependency, because aglio isn't part of the actual codebase.

#### How to test
Run `grunt test` and make sure everything we expect to pass passes.
Use the app and make sure nothing is broken
